starknet_py/net/schemas/utils.py

```diff
@@ -0,0 +1,11 @@
+def replace_sender_address_with_contract_address(data: dict):
+    data["contract_address"] = data.get("contract_address") or data.get(
+        "sender_address"
+    )
+
+    if data["contract_address"] is None:
+        raise ValueError(
+            "Missing field `contract_address` or `sender_address` for InvokeTransactionSchema."
```

Comment:
Do we want to be mentioning specific `InvokeTransactionSchema` in a function from other module?

Reply:
Good point. I think we do. It is used in both rpc and gateway `InvokeTransactionSchema`.